dev/tools/controllerbuilder/pkg/options/generateoptions.go

```diff
@@ -27,6 +27,6 @@ func (o *GenerateOptions) InitDefaults() {
 
 func (o *GenerateOptions) BindPersistentFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&o.ProtoSourcePath, "proto-source-path", o.ProtoSourcePath, "path to (compiled) proto for APIs")
-	cmd.PersistentFlags().StringVarP(&o.APIVersion, "version", "v", o.APIVersion, "the KRM API version. used to import the KRM API")
+	cmd.PersistentFlags().StringVarP(&o.APIVersion, "apiVersion", "v", o.APIVersion, "the KRM API version. used to import the KRM API")
```

Comment:
Nit: this probably should be `api-version`, to be consistent with `proto-source-path`.

Also ... do we need to change generate.sh?

Reply:
Good point! Done